sct.py

```diff
@@ -191,5 +193,36 @@ def conf_docs(output_format):
         click.secho(SCTConfiguration().dump_help_config_yaml())
 
 
+@click.group(help="Group of commands for investigating testrun")
+def investigate():
+    pass
+
+
+@investigate.command('show-logs', help="Show logs collected for testrun filtered by test-id")
+@click.argument('test_id')
+def show_log(test_id):
+    x = PrettyTable(["Log type", "Link"])
+    x.align = "l"
+    files = list_logs_by_test_id(test_id)
+    for log in files:
+        x.add_row([log["type"], log["link"]])
+    click.echo(x.get_string(title="Log links for testrun with test id {}".format(test_id)))
+
+
+@investigate.command('show-monitor', help="Show link to prometheus data snapshot")
+@click.argument('test_id')
+def show_monitor(test_id):
```

Comment:
what's the point of a separate command ? the monitor snapshot logs would be show-logs.

also I think 'hydra investigate logs' is enough

Reply:
Need discuss with Bentsi, but  show monitor should be more complex in future. it should download snapshot, restore data from archive and run monitoring node with this data. This is the first stage